Library/Homebrew/unversioned_cask_checker.rb

```diff
@@ -78,8 +83,18 @@ def all_versions
 
         installer.extract_primary_container(to: dir)
 
-        info_plist_paths = apps.flat_map do |app|
-          top_level_info_plists(Pathname.glob(dir/"**"/app.source.basename/"Contents"/"Info.plist")).sort
+        info_plist_paths = []
+
+        apps.flat_map do |app|
+          top_level_info_plists(Pathname.glob(dir/"**"/app.source.basename/"Contents"/"Info.plist"))
+            .sort
+            .each { |match| info_plist_paths.push(match) }
+        end
+
+        qlplugins.flat_map do |qlplugin|
+          top_level_info_plists(Pathname.glob(dir/"**"/qlplugin.source.basename/"Contents"/"Info.plist"))
+            .sort
+            .each { |match| info_plist_paths.push(match) }
         end
```

Comment:
1. @bevanjkay, I propose to shorten this a bit to:
```suggestion
        info_plist_paths = (apps + qlplugins).flat_map do |app|
          top_level_info_plists(Pathname.glob(dir/"**"/app.source.basename/"Contents"/"Info.plist")).sort
        end
```
2. I assume the same change should be applied to `guess_cask_version`.
3. @reitermarkus, do you know why we're `.sort`-ing inside the `do/end` and not the result (`end.sort`)? Is that `.sort` necessary?

Reply:
@maxim-belkin thanks for the refactor. I'm fairly new to ruby so good to know that syntax is possible

Reply:
> I'm fairly new to ruby

not a problem. Many Homebrew maintainers are in a perpetual state of being new to Ruby :)